src/Metrics/Metrics.hs

```diff
@@ -82,10 +66,10 @@ registerProjectMetrics = ProjectMetrics
   <*> register (vector "project" (gauge (Info "hoff_project_merge_train_size"
                                                 "Number of pull requests currently in the queue (merge train)")))
 
-incProjectMergedPR :: (MonadMonitor m, MonadIO m) => ProjectMetrics -> ProjectLabel -> m ()
+incProjectMergedPR :: ProjectMetrics -> ProjectLabel -> IO ()
```

Comment:
Just filling in `IO` instead of those two class constraints made it easier as well.